src/containers/ToolboxSubject/ToolboxSubjectContainer.tsx

```diff
@@ -0,0 +1,186 @@
+      window.scrollTo({
+        top: positionFromTop - 185,
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  const subject = data.subject;
+
+  const topics = subject.topics?.map((topic: GQLTopic) => {
+    return {
+      ...topic,
+      label: topic.name,
+      selected: topic.id === topicList[0],
+      url: toTopic(subject.id, topic.id),
+    };
+  });
+
+  const onSelectTopic = (
+    e: React.MouseEvent<HTMLAnchorElement>,
+    index: number,
+    id?: string,
+  ) => {
+    e.preventDefault();
+    if (id) {
+      const topic = subject.allTopics?.find(
+        (topic: GQLTopic) => topic.id === id,
+      );
+      if (topic) {
+        if (index === 0) {
+          setSelectedTopics([topic.id]);
+        } else if (index > 0) {
+          const updatedSelectedTopics = selectedTopics.slice(0, index + 1);
+          updatedSelectedTopics[index] = id;
+          setSelectedTopics(updatedSelectedTopics);
+        }
+        const path = parseAndMatchUrl(e.currentTarget?.href, true);
+        history.replace({
+          pathname: path?.url,
+        });
+      }
+    }
+  };
+
+  const TopicBoxes = () => (
+    <>
+      {selectedTopics.map((topic: string, index: number) => {
+        return (
+          <div key={index} ref={refs[index]}>
+            <ToolboxTopicWrapper
+              subjectId={subject.id}
+              topicId={topic}
+              locale={locale}
+              onSelectTopic={onSelectTopic}
+              topicList={topicList}
+              index={index}
+            />
+          </div>
+        );
+      })}
+    </>
+  );
+
+  if (!topics) {
+    return null;
+  }
+
+  return (
+    <>
+      <Helmet>
+        <title>
+          {htmlTitle(data.subject?.name, [t('htmlTitles.titleTemplate')])}
+        </title>
+      </Helmet>
+      <OneColumn className={''}>
+        <ToolboxInfo
+          topics={topics}
+          onSelectTopic={(e: React.MouseEvent<HTMLElement>, id?: string) =>
+            onSelectTopic(e as React.MouseEvent<HTMLAnchorElement>, 0, id)
+          }
+          title={getSubjectLongName(subject.id, locale) || subject.name}
+          introduction={t('htmlTitles.toolbox.introduction')}
+        />
+        <TopicBoxes />
+        <SubjectBanner image={''} negativeTopMargin={!topics} />
+      </OneColumn>
+    </>
+  );
+};
+
+ToolboxSubjectContainer.getDocumentTitle = getDocumentTitle;
+
+ToolboxSubjectContainer.willTrackPageView = (
+  trackPageView: (item: Props) => void,
+  currentProps: Props,
+) => {
+  if (currentProps.data.subject) {
+    trackPageView(currentProps);
+  }
+};
+
+ToolboxSubjectContainer.getDimensions = (props: Props) => {
+  const { data, locale, topicList } = props;
+  const topicPath = topicList.map(t =>
+    data.subject.allTopics.find(topic => topic.id === t),
+  );
+  const longName = getSubjectLongName(data.subject?.id, locale);
+
+  return getAllDimensions({
+    subject: data.subject,
+    topicPath,
+    filter: longName,
+  });
```

Comment:
Ser at dimensions som regnes ut ikkje har med dim 8, som den skulle hatt for alle emner som vises. Så om du klikker på et emne må du sende inn isArticle som parameter.